Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
OK I found alpha versions are included in some previous commits, for example 75f72923eb6fe4fc6d4d1ad9a2669301274267da, so ```devel``` version is not reliable. Also, updating the devel version may take hours or even days in Homebrew, during which things may break.

@ilovezfs's suggestion is possible. Currently in python3.rb, ```#{xy}``` is used only after ```make``` is invoked, while I guess it's dangerous to assume it will never change in the future. My proposal is indeed lengthy and verbose yet robust, and only upstream changes can break it.